<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowShop</class>
 <widget class="QMainWindow" name="MainWindowShop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelCustomer">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="labelIntro">
        <property name="text">
         <string>Please complete the required data to record your purchase.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelDate">
        <property name="text">
         <string>Date: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLabel" name="labelDateTime">
        <property name="text">
         <string>Display the current date and time.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="labelType">
        <property name="text">
         <string>Select type of item you want:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="labelSize">
        <property name="text">
         <string>Select size of item you want: </string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="labelNumber">
        <property name="text">
         <string>Select number of item you want: </string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="comboBoxType">
        <item>
         <property name="text">
          <string>Tie-Dye T-Shirt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom Printed T-Shirt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Made Branded T-Shirt</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="comboBoxSize">
        <item>
         <property name="text">
          <string>Small(S)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium(M)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Large(L)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Extra Large(XL)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="pushButtonAdd">
        <property name="text">
         <string>Add Item</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/files/images/accept.png</normaloff>:/files/images/accept.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelCustomerName">
        <property name="text">
         <string>John Doe</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCloseApp">
      <property name="text">
       <string>Done Close App</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/files/images/cancel.png</normaloff>:/files/images/cancel.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionSave_File"/>
    <addaction name="actionSave_File_As"/>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionSelect_None"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionToolbar_top"/>
    <addaction name="actionToolbar_bottom"/>
    <addaction name="actionToolbar_right"/>
    <addaction name="actionToolbar_left"/>
    <addaction name="separator"/>
    <addaction name="actionToolbar_movable"/>
    <addaction name="actionToolbar_floatable"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_File"/>
   <addaction name="actionSave_File"/>
   <addaction name="actionOpen_File"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionSelect_All"/>
   <addaction name="actionSelect_None"/>
  </widget>
  <action name="actionNew_File">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/new.png</normaloff>:/files/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/disk.png</normaloff>:/files/images/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/open.png</normaloff>:/files/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/cancel.png</normaloff>:/files/images/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/copy.png</normaloff>:/files/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy </string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/cut.png</normaloff>:/files/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/paste.png</normaloff>:/files/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/selectall.png</normaloff>:/files/images/selectall.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionToolbar_top">
   <property name="text">
    <string>Toolbar top</string>
   </property>
  </action>
  <action name="actionToolbar_bottom">
   <property name="text">
    <string>Toolbar bottom</string>
   </property>
  </action>
  <action name="actionToolbar_right">
   <property name="text">
    <string>Toolbar right</string>
   </property>
  </action>
  <action name="actionToolbar_left">
   <property name="text">
    <string>Toolbar left</string>
   </property>
  </action>
  <action name="actionToolbar_movable">
   <property name="text">
    <string>Toolbar movable</string>
   </property>
  </action>
  <action name="actionToolbar_floatable">
   <property name="text">
    <string>Toolbar floatable</string>
   </property>
  </action>
  <action name="actionSave_File_As">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/saveas.png</normaloff>:/files/images/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save File As</string>
   </property>
  </action>
  <action name="actionSelect_None">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/files/images/selectnone.png</normaloff>:/files/images/selectnone.png</iconset>
   </property>
   <property name="text">
    <string>Select None</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
